src/cgame/default/views/PlayerModelView.c

```diff
@@ -185,7 +191,9 @@ static void render(View *self, Renderer *renderer) {
 		glMatrixMode(GL_PROJECTION);
 		glLoadIdentity();
 
-		glOrtho(0, cgi.context->window_width, cgi.context->window_height, 0, -1, 1);
+		Matrix4x4_FromOrtho(&mat, 0.0, cgi.context->window_width, cgi.context->window_height, 0.0, -1.0, 1.0);
+
+		glLoadMatrixf((GLfloat *) mat.m);
 
 		glMatrixMode(GL_MODELVIEW);
 		glLoadIdentity();
```

Comment:
Juicy. There is also some slop in this View for calculating the matrix of the linked models (head, torso, weapon). Not sure if any of this `matrix4x4_t` cleanup will allow us to remove that (doubtful). But just so you're aware of it..